exercises/roman-numerals/introduction.md

```diff
@@ -0,0 +1,59 @@
+# Description
+
+Today, most people in the world use Arabic numbers.
+But if you travelled back two thousand years, you'd find that most Europeans were using Roman numerals instead.
+
+To write a Roman numeral we use the following Latin letters, each of which has a value:
+
+| M    | D   | C   | L  | X  | V | I |
+|------|-----|-----|----|----|---|---|
+| 1000 | 500 | 100 | 50 | 10 | 5 | 1 |
+
+A Roman numeral is a sequence of these letters, and its value is the sum of the letters' values.
+For example, `XVIII` has the value 18 (`10 + 5 + 1 + 1 + 1 = 18`).
+
+There's one rule that makes things tricker though, and that's that **the same letter cannot be used more than three times in succession**.
```

Comment:
I expect the subtraction construct to be easier to contextualize when it is already known that letters are generally ordered in order of decreasing value, so I'd put the note about order between these paragraphs.

As of yet I have no better idea than

```suggestion
For example, `XVIII` has the value 18 (`10 + 5 + 1 + 1 + 1 = 18`).

Order matters in Roman numerals.
Letters (and the special compounds given below) must be ordered by decreasing value from left to right.

There's one rule that makes things trickier though, and that's that **the same letter cannot be used more than three times in succession**.
```